cmd/cmd/show.go

```diff
@@ -55,7 +58,17 @@ var showCmdHelp = `
 Use " show [command] --help" for more information about a command
 `
 
-var showCmdValidArgs = []string{"api", "carbonapp", "endpoint", "inboundendpoint", "proxyservice", "sequence", "task", "help"}
+var showCmdValidArgs = []string{
+	"api",
+	"carbonapp",
+	"endpoint",
+	"inboundendpoint",
+	"proxyservice",
+	"sequence",
+	"task",
+	"data-service",
```

Comment:
This is different from `proxyservice`. Can we make it consistent?

Reply:
What shall we use? I'm in favor of using a hyphen.

Reply:
Since dataservice is a term specific to integration, It is ok to use without the hyphen.